pkg/ccl/multiregionccl/testutils.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package multiregionccl
+
+import (
+	"context"
+	gosql "database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/base"
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
+)
+
+// CreateTestMultiRegionCluster creates a test cluster with numServers number of
+// nodes with the provided testing knobs applied to each of the nodes. Every
+// node is placed in its own locality, named "us-east1", "us-east2", and so on.
+func CreateTestMultiRegionCluster(
```

Comment:
nit: testutils should probably go in a separate package and/or be called `TestingCreateMultiRegionCluster`

Reply:
(i'm not happy that the "testing" package gets imported in main though, so rather this live in a separate package or return an error)